doc/sphinxext/gallery_generator.py

```diff
@@ -236,6 +238,18 @@ def thumbfilename(self):
         pngfile = self.modulename + "_thumb.png"
         return pngfile
 
+    @property
+    def apiname(self):
+        with open(op.join(self.target_dir, self.pyfilename), "r") as file:
+            regex = r"az\.(plot\_[a-z_]+)\("
+            name = re.findall(regex, file.read())
+        apitext = name[0] if len(name) > 0 else ""
+        return (
+            "`" + apitext + " <../../generated/arviz." + apitext + ">`_"
```

Comment:
.format can be used here instead of adding strings

Reply:
Okay, I'll add it.